src/traccuracy/track_errors/_ctc.py

```diff
@@ -94,6 +98,17 @@ def get_edge_errors(matched_data: "Matched"):
     node_mapping_first = np.array([mp[0] for mp in node_mapping])
     node_mapping_second = np.array([mp[1] for mp in node_mapping])
 
+    # intertrack edges = connection between parent and daughter
+    for graph in [comp_graph, gt_graph]:
+        # Set to False by default
+        graph.set_edge_attribute(list(graph.edges()), EdgeAttr.INTERTRACK_EDGE, False)
+
+        for parent in graph.get_divisions():
```

Comment:
@msschwartz21 we should probably also look at merges, which are also intertrack edges, so any edge whose target node has another incoming edge.